cli/src/cluster_query.rs

```diff
@@ -382,6 +383,25 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         ])
                         .default_value("stake")
                         .help("Sort order (does not affect JSON output)"),
+                )
+                .arg(
+                    Arg::with_name("keep_unstaked_delinquents")
+                        .long("keep-unstaked-delinquents")
+                        .takes_value(false)
+                        .help("Don't discard unstaked, delinquent validators")
+                )
+                .arg(
+                    Arg::with_name("delinquent_slot_distance")
+                        .long("delinquent-slot-distance")
+                        .takes_value(true)
+                        .value_name("SLOT_DISTANCE")
+                        .validator(is_slot)
+                        .help(
+                            concatcp!(
```

Comment:
TIL! Darn, could have used this recently

Reply:
Ha!  I thought I was going to be clever and use `concat!()` which I learned from your `storage-proto` `build.rs`, but it blew up on the constant.  Crates.io to the rescue!